ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles.json

```diff
@@ -319,7 +319,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "RoleName": "specific-access-legal-operations",
     "ReadOnly": "N",
-    "AccessProfiles": "caseworker-publiclaw-judiciary"
+    "AccessProfiles": "caseworker-publiclaw-judiciary,caseworker-publiclaw-courtadmin"
```

Comment:
think this just needs to be caseworker-publiclaw-judiciary! but other than that all good 😄